usagov/vote-gov-drupal
# Bug/vote 274/mail time string translations

## Jira ticket
https://cm-jira.usa.gov/browse/VOTE-274

## Description
Add the mail deadline string translations to the .po files
Translate the times in the double field deadlines using JSON string replace

## Deployment and testing

### Pre-deploy
1. lando retune

### Post-deploy

1. Edit one of the state pages to use the specific registration deadlines. Set the times for both double fields to 12:00 PM. 

### QA/Test

1. Verify that "Must be postmarked by" is translated for each language
2. Verify that "Must be received by" is translated for each language (need to change this in the editor for the state page)
3. Verify that the times are translated for the applicable languages. Some languages use the same time as English.

## Checklist for the Developer




## Checklist for the Peer Reviewers
